python/FlexSource.py

```diff
@@ -112,10 +130,10 @@ def __iq_data_received(self, iq_stream, data):
         except Exception as err:
             print err
 
-    """ # If uncommenting, also see the += line in def start(self)
+    """# If uncommenting, also see the += line in def start(self)
```

Comment:
If this section is useful for debugging, we could just add a debug flag instead of remembering to comment/uncomment out stuff.

i.e.
```python
if self._debug
    self.pan_adapter.PropertyChanged += self.__property_changed
```